modules/dataframe/src/properties/dataframecolormapproperty.cpp

```diff
@@ -90,6 +90,9 @@ DataFrameColormapProperty::DataFrameColormapProperty(std::string identifier,
 void DataFrameColormapProperty::createOrUpdateProperties(
     std::shared_ptr<const DataFrame> dataframe) {
     if (!dataframe || dataframe->getNumberOfColumns() <= 1) return;
+
+    if (dataframe_ == dataframe) return;
```

Comment:
Since dataframe_ can be mutated above in the network, this does not mean that it hasn't changed, hence this seems quite error prone...